libs/util/kafka/src/types/kafka.types.ts

```diff
@@ -0,0 +1,13 @@
+import { IHeaders, KafkaMessage as FullKafkaMessage } from "kafkajs";
+
+export type Json = Record<string, unknown>;
+
+export interface DecodedKafkaMessage {
+  key?: string | Json;
+  value: string | Json | null;
```

Comment:
Are you sure that value can be null? Even if so, can't it be undefined then?

Reply:
From a schema point of view, value cannot ever be optional and `null` would be a valid value in Avro and Json. Declaring this as undefined might serde processes.

Reply:
interface updated to be more consistent